src/amo/components/GetFirefoxButton/index.js

```diff
@@ -72,7 +73,9 @@ export type GetDownloadLinkParams = {|
   _getDownloadCampaign?: typeof getDownloadCampaign,
   addon?: AddonType,
   clientApp?: string,
-  overrideQueryParams?: {| [name: string]: string | null |},
+  // Use `undefined` to force the fallback value, and `null` to remove the
+  // query param in the query string.
+  overrideQueryParams?: {| [name: string]: string | null | typeof undefined |},
```

Comment:
These seems like magic values - you have to add a comment everywhere it's used so we know it's magic...